README.md

```diff
@@ -58,27 +58,28 @@ dependency)
 make generate
 ```
 
-3. Edit the code in your favorite IDE
+You should use the included build image to produce a consistent set of generated file, e.g.
 
-4. Format, vet and lint the code
+```sh
+docker run -v $(pwd):/go-control-plane gcr.io/istio-testing/go-control-plane-ci:05-09-2019 make generate
+```
+
+Format the code:
 
 ```sh
 make format
-make check
 ```
 
-5. Build and test
+__NOTE__: you may need to apply a small patch to correct imports in the generate files:
```

Comment:
The patch is programmatic using sed. The underlying issue is in the code that is run by protoc as a plugin (protoc-gen-validate). It seems to ignore the custom import paths we have here, and it's bit too difficult to nail it down, since it seems to come from another framework (https://github.com/lyft/protoc-gen-star).